test/kata3.numberToReversedDigits.test.js

```diff
@@ -1,5 +1,8 @@
 const { numberToReversedDigits } = require("../src");
 
 describe("numberToReversedDigits", () => {
-  test("returns a reversed array of the number's digits", () => {});
-});
\ No newline at end of file
+  test("returns a reversed array of the number's digits", () => {
```

Comment:
The test requirements are not quite correct. It should be asserting that the function returns an array of numbers when passed an argument with the `number` datatype, not a `string`. 

```js
expect(numberToReversedDigits(12345)).toEqual([5, 4, 3, 2, 1])
```

This would have required a more challenging solution by performing the following operations: 
1. Convert the `number` argument to a string
2. Convert the string to an array
3. Reverse the array